src/extension.ts

```diff
@@ -64,6 +64,13 @@ export async function activate(context: ExtensionContext): Promise<XMLExtensionA
 
   languageClient = await startLanguageClient(context, serverOptions, logfile, externalXmlSettings, requirementsData);
 
+  const onDidGrantWorkspaceTrust = (workspace as any).onDidGrantWorkspaceTrust;
```

Comment:
The `onDidGrantWorkspaceTrust` should be managed in xmlClient at https://github.com/redhat-developer/vscode-xml/blob/ed8e6bb8c4bc686144870d58f7dfb7f9dd4715e5/src/client/xmlClient.ts#L105